### DISCUSSION TITLE
Let's Connect

### DISCUSSION THREAD

**Discussion by raselldev** [depapp/IndoGitHubers#15] [General]
Ayok kita saling follow agar masuk kriteria 25 followers!!!

**Comment by depapp** [comment]
> Ayok kita saling follow agar masuk kriteria 25 followers!!!

followed, bro 🤜🏻

**Reply by raselldev** [reply]
makasih bro

**Comment by AsadSaleh** [comment]
Yes. Tolong follow saya. Biar masuk list, hehe 😆

**Comment by mazipan** [comment]
If you are following me because of this thread, feel free to mention me in this below thread.

I will try to following you back.
